extensions/wikia/Search/classes/IndexService/Metadata.php

```diff
@@ -1,48 +1,49 @@
 <?php
 /**
  * Class definition for \Wikia\Search\IndexService\Metadata
+ *
  * @author relwell
  */
 namespace Wikia\Search\IndexService;
 /**
  * Allows us to access mediawiki-dependent metadata
+ *
  * @author relwell
  * @package Search
  * @subpackage IndexService
  */
-class Metadata extends AbstractService
-{
+class Metadata extends AbstractService {
 	/**
 	 * Queries the API for article metadata
+	 *
+	 * @throws \WikiaException
 	 * @see \Wikia\Search\IndexService\AbstractService::execute()
 	 * @return array
 	 */
 	public function execute() {
-		$result = array();
-		
-		$service = $this->getService();
-	
-		if (! $this->getService()->isOnDbCluster() ) {
-			return array();
+		$result = [];
+
+		if ( !$this->getService()->isOnDbCluster() ) {
+			return [];
 		}
-		
+
 		$pageId = $this->currentPageId;
 		if ( $pageId === null ) {
 			throw new \WikiaException( 'A pageId-dependent indexer service was executed without a page ID queued' );
 		}
 
 		$data = $this->service->getApiStatsForPageId( $pageId );
-		if( isset( $data['query']['pages'][$pageId] ) ) {
+		if ( isset( $data['query']['pages'][$pageId] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "query" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)